extra-devel/musl/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=musl
+PKGSEC=devel
+BUILDDEP="devel-base"
+PKGDES="Lightweight implementation of C standard library"
+
+NOSTATIC=0
+ABSPLITDBG=0
```

Comment:
Why?

Reply:
There is no `BuildID[sha1]` for several `.o` files, so Autobuild3 will fail without `ABSPLITDBG=0`.

Reply:
![musl-libc](https://user-images.githubusercontent.com/12680076/224340309-752739a7-21d2-4036-a19b-4c7cb3f51cc3.png)
musl provides libc.so.

Reply:
Then, in this case, you should see if flags defined by Autobuild3 were passed into the build system.